### PR TITLE
fix: pro init 流程完善

### LINKED ISSUE #72: 🐛 [Bug]: 在执行"tiny init pro"时 ,选择服务器技术栈"Spring Cloud", 会报"项目模板创建失败"
### Version

tiny v1.0.1

### node-version

v18.16.1

### Link to minimal reproduction

_No response_

### Step to reproduce

8bb3197b05357445f1af2f4d6a06eba
image

### What is expected

建议参考若依开发平台, 部署过程清晰
https://doc.ruoyi.vip/ruoyi-vue/document/hjbs.html#准备工作
后端可以不以增加SpringBoot版本 

### What is actually happening

_No response_

### Any additional comments (optional)

_No response_

### PR DESCRIPTION
**Pull request by wjiangwang**
# PR
修改 tiny init pro 流程中
1. 默认选择不对接服务端（用户全部选择默认选项，可直接运行）
2. 删除对接服务器中 springcloud 和 nextjs 选项
3. 以用户输入的项目名（默认tiny-pro），在当前路径下创建新的文件夹来存放项目。不平铺
4. 选择服务端后，增加强提示和说明文档链接。让用户确认数据库服务是否安装好
image

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-cli/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/toolkits/pro/src/lib/init.ts [hunk 1]
```diff
@@ -51,13 +50,23 @@ const getProjectInfo = (): Promise<ProjectInfo> => {
...
-      default: ServerFrameworks.EggJs,
+      default: ServerFrameworks.Skip,
       prefix: '*',
     },
+    {
+      type: 'list',
+      name: 'serverConfirm',
+      message:
+        '请您确保已经安装相关数据库服务（参考文档：https://www.opentiny.design/tiny-cli/docs/toolkits/pro）：',
+      choices: [
```
Comment (hello-yezi) [inline packages/toolkits/pro/src/lib/init.ts:65]: https://www.opentiny.design/tiny-cli/docs/toolkits/pro 文档 需要同步更新：
image

File: packages/toolkits/pro/src/lib/init.ts [hunk 2]
```diff
@@ -51,13 +50,23 @@ const getProjectInfo = (): Promise<ProjectInfo> => {
...
       ],
-      default: ServerFrameworks.EggJs,
+      default: ServerFrameworks.Skip,
       prefix: '*',
     },
+    {
+      type: 'list',
+      name: 'serverConfirm',
+      message:
+        '请您确保已经安装相关数据库服务（参考文档：https://www.opentiny.design/tiny-cli/docs/toolkits/pro）：',
```
Comment (hello-yezi) [inline packages/toolkits/pro/src/lib/init.ts]: 1.文案建议修改：请确保已安装数据库服务（xxx）
2.链接地址写成https://www.opentiny.design/tiny-cli/docs/toolkits/pro#database 会不会好一些？

File: packages/toolkits/pro/src/lib/init.ts [hunk 3]
```diff
@@ -18,13 +18,12 @@ const NG_TEMPLATE_PATH = 'tinyng';
...
  */
 const getProjectInfo = (): Promise<ProjectInfo> => {
-  const basename = path.basename(utils.getDistPath());
   const question: QuestionCollection<ProjectInfo> = [
     {
       type: 'input',
       name: 'name',
       message: '请输入项目名称：',
-      default: basename,
+      default: 'tiny-pro',
```
Comment (hello-yezi) [inline packages/toolkits/pro/src/lib/init.ts]: tiny-pro建议定义为全局变量

File: packages/toolkits/pro/src/lib/init.ts (hunk 1 above)
Comment (wjiangwang) [reply packages/toolkits/pro/src/lib/init.ts:65]: 已修改。待docs仓库合并后发布

File: packages/toolkits/pro/src/lib/init.ts (hunk 2 above)
Comment (wjiangwang) [reply packages/toolkits/pro/src/lib/init.ts]: 已修改
image

File: packages/toolkits/pro/src/lib/init.ts (hunk 3 above)
Comment (wjiangwang) [reply packages/toolkits/pro/src/lib/init.ts]: 已修改

**Review by hello-yezi — APPROVED** [review]

**Comment by hello-yezi** [comment]
LGTM

**Review by h-ivy — APPROVED** [review]
ok

**Pull request merged**